dev-cpp/notcurses/notcurses-2.2.1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 nick black
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS="cmake"
+inherit cmake-multilib
+
+DESCRIPTION="Blingful TUIs and character graphics"
+HOMEPAGE="https://notcurses.com"
+SRC_URI="https://github.com/dankamongmen/notcurses/archive/v2.2.1.tar.gz"
+
+LICENSE="APACHE-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="app-text/pandoc dev-libs/unistring media-video/ffmpeg sys-libs/readline"
+RDEPEND="${DEPEND}"
+BDEPEND=""
+
+src_prepare() {
+	cmake_src_prepare
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DUSE_DOCTEST=OFF
+		-DUSE_QRCODEGEN=OFF
+		-DUSE_PANDOC=OFF
```

Comment:
You have `app-text/pandoc` in `DEPEND` but yet disable pandoc usage. Why?

Reply:
Brain freeze. I'll make this conditional based on the user selecting `doc`, as you suggested.